core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -1497,11 +1492,13 @@ public void printInfo(boolean includeIndexDetails) throws IOException {
 
     @Override
     public void setInterruptFlag(AtomicBoolean flag) {
-      if (deepCopy)
+      if (deepCopy) {
         throw new RuntimeException("Calling setInterruptFlag on a deep copy is not supported");
+      }
 
-      if (deepCopies.size() != 0)
+      if (!deepCopies.isEmpty()) {
```

Comment:
Unrelated changes, but nice changes to see anyway. I sometimes have my IDE automatically add missing braces for single-statement blocks, but the downside is that they can clutter the review (same with the unrelated change to convert this to `isEmpty()`). I don't think there' a problem here... the changes in this PR are simple enough to understand, and these are nice changes to see.